rust/noosphere-core/src/authority/author.rs

```diff
@@ -20,7 +20,9 @@ use super::generate_capability;
 /// else read-only access (to all other spheres).
 #[derive(PartialEq, Eq, Debug, Clone)]
 pub enum Access {
```

Comment:
Well `SimulatedSphereAccess` is actually domain-specific to that helper. So, in tests where you use that helper, you need to use `SimulatedSphereAccess` (because it configures what the helper does when it produces a sphere for you).

Reply:
Context: generating a non-MemoryStore DB-backed sphere in tests https://github.com/subconsciousnetwork/noosphere/pull/623/files#diff-da1f415011687e703e572ced0f53a376dc3707ee9e6ccedf24eddea48c72a8a5R55